src/components/spreadsheet/equipment-table.js

```diff
@@ -70,8 +68,9 @@ export const EquipmentTable = ({
             editErrors: {},
             dynamicValidation: {},
             isEditing: topPinnedData ? true : false,
```

Comment:
Could you add `lastEditedField` to the default state of the context so we get a full list of all context variables ?

Reply:
Done